stockfish/models.py

```diff
@@ -550,6 +550,60 @@ def set_depth(self, depth_value: int = 2) -> None:
         """
         self.depth = str(depth_value)
 
+    def get_what_is_on_square(self, square: str) -> str:
+        """Returns what is on the specified square.
+
+        Args:
+            square:
+                The coordinate of the square in question. E.g., e4.
+
+        Returns:
+            One of: "P", "p", "N", "n", "B", "b", "R", "r", "Q", "q",
+            "K", "k", or " ".
+            Uppercase represents a white piece, lowercase represents a black
+            piece, and a single space represents an empty square.
+        """
+
+        file_letter = square[0].lower()
+        rank_num = int(square[1])
+        if (
+            len(square) != 2
+            or file_letter < "a"
+            or file_letter > "h"
+            or square[1] < "1"
+            or square[1] > "8"
+        ):
+            raise ValueError(
+                "square argument to the get_what_is_on_square function isn't valid."
+            )
+        rank_visual = self.get_board_visual().splitlines()[17 - 2 * rank_num]
+        return rank_visual[2 + (ord(file_letter) - ord("a")) * 4]
+
+    def will_move_be_a_capture(self, move_value: str) -> str:
```

Comment:
Maybe it would be better to return enum?

```python
from enum import Enum

class Capture(Enum):
    DIRECT_CAPTURE = "direct capture"
    EN_PASSANT = "en passant"
    NO_CAPTURE = "no capture"
```